charts/exerciseset2/templates/exerciseSet.yaml

```diff
@@ -0,0 +1,95 @@
+---
+apiVersion: kubeteach.geberl.io/v1alpha1
+kind: ExerciseSet
+metadata:
+  name: set2
+spec:
+  taskDefinitions:
+    - name: task01
+      taskDefinitionSpec:
+        taskSpec:
+          title: "Create namespace"
+          description: "Create a new namespace with the name kubeteach"
+          helpURL: "https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/"
+        taskCondition:
+          - apiVersion: v1
+            kind: Namespace
+            name: "kubeteach"
+        points: 5
+    - name: task02
+      taskDefinitionSpec:
+        taskSpec:
+          title: "Create a Persistent Volume Claim"
+          description: "Create a Persistent Volume Claim (PVC) in the cluster with the name myfirstpvc in the task01's namespace, request 1Gi storage with RWO access mode"
+        requiredTaskName: task01
+        taskCondition:
+          - apiVersion: v1
+            kind: PersistentVolumeClaim
+            name: "myfirstpvc"
+            namespace: kubeteach
+            resourceCondition:
+              - field: "spec.resources.requests.storage"
+                operator: "eq"
+                value: "1Gi"
+        points: 5
+    - name: task03
+      taskDefinitionSpec:
+        taskSpec:
+          title: "Create a Persistent Volume"
+          description: "Create a Persistent Volume (PV) in the cluster with the name myfirstpv in the task01's namespace, spec 1Gi of capacity with RWO access mode. Moreover, use the hostpath.path /opt/volume/nginx and specify the claimRef for task02's PVC"
+        requiredTaskName: task02
+        taskCondition:
+          - apiVersion: v1
+            kind: PersistentVolume
+            name: "myfirstpv"
+            resourceCondition:
+              - field: "spec.capacity.storage"
+                operator: "eq"
+                value: "1Gi"
+              - field: "spec.claimRef.name"
+                operator: "eq"
+                value: "myfirstpvc"
+              - field: "status.phase"
+                operator: "eq"
+                value: "Bound"
+    - name: task04
+      taskDefinitionSpec:
+        taskSpec:
+          title: "Create a Pod"
+          description: "Create a Pod pod1 with the nginx:latest image, assign the pod the volume and specify the PVC"
```

Comment:
I think we should add also a volumeMount to the task, otherwise the volume cant be used.